src/framework/uicomponents/qml/MuseScore/UiComponents/StyledMenuItem.qml

```diff
@@ -1,89 +1,153 @@
-import QtQuick 2.0
-import QtQuick.Controls 2.12
-import QtQuick.Layouts 1.3
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
 
+import MuseScore.UiComponents 1.0
 import MuseScore.Ui 1.0
 
-MenuItem {
+ListItemBlank {
     id: root
 
-    implicitHeight: 30
-    implicitWidth: 220
+    defaultColor: ui.theme.accentColor
+    enabled: Boolean(modelData) && Boolean(modelData.enabled)
 
-    property var hintIcon: checkable && checked ? IconCode.TICK : IconCode.NONE
-    property string shortcut: ""
+    isSelected: privateProperties.showedSubMenu != undefined || (privateProperties.hasIcon && privateProperties.isSelectable && privateProperties.isSelected)
 
-    background: Rectangle {
-        id: background
+    property var modelData
+    property bool reserveSpaceForInvisibleItems: true
```

Comment:
Can you explain please what is this needed for?

Reply:
By default, I reserve space for invisible items like shortcuts, icons, etc.
But this is not necessary for some popups, for example, page view control.
![image](https://user-images.githubusercontent.com/10116828/111273211-35944000-863c-11eb-8656-8b645b9ca6ec.png)